.github/workflows/test_s3.yml

```diff
@@ -42,3 +47,20 @@ jobs:
           sleep 5
           mcli alias set $MINIO_ALIAS $MINIO_URL $MINIO_ROOT_USER $MINIO_ROOT_PASSWORD
           mcli admin user svcacct add --access-key $MINIO_ACCESS_KEY --secret-key $MINIO_SECRET_KEY $MINIO_ALIAS $MINIO_ROOT_USER
+
+      - name: Create a bucket
+        run: |
+          mcli mb $MINIO_ALIAS/$MINIO_BUCKET
+
+      - name: Install
+        run: |
+          pip install --upgrade pip
+          pip install -e '.[testing]'
```

Comment:
Why do we need to run this in "editable" mode?

Reply:
We probably don't.